client/e2e/operations.spec.ts

```diff
@@ -0,0 +1,101 @@
+import { test, expect } from "@playwright/test";
+
+// Annotate entire file as serial.
+test.describe.configure({ mode: "serial" });
+
+test.beforeEach(async ({ page }, testInfo) => {
+  // eslint-disable-next-line no-console
+  console.log(`Running ${testInfo.title}`);
+  // reset db and confirm there are no errors setting up the test data
+  await page.goto("http://localhost:8000/api/registration/test-setup");
```

Comment:
I think we'll need to parameterize these URLs in order for them to work in Openshift environments...?

Reply:
... maybe? Do we normally run e2e tests in Openshift, or just locally/in github actions?

Reply:
fair point. I don't know if we do on CAS (haven't been paying attention). I assumed so but that's entirely baseless. Any of the devopsy people (e.g., @dleard  , @Sepehr-Sobhani  or @mikevespi) able to shed light on this?

Reply:
only in gh actions